internal/avs/external_eval_assistant.go

```diff
@@ -48,8 +49,51 @@ func (eea *ExternalEvalAssistant) ProvideParentId(_ internal.ProvisioningParamet
 	return eea.avsConfig.ParentId
 }
 
-func (eea *ExternalEvalAssistant) ProvideTags() []*Tag {
-	return eea.avsConfig.ExternalTesterTags
+func (eea *ExternalEvalAssistant) ProvideTags(pp internal.ProvisioningParameters) []*Tag {
+	var Tags []*Tag
+	Tags = append(Tags, &Tag{
+		Content:      pp.ErsContext.TenantID,
+		TagClassId:   320334701,
```

Comment:
All TagClasdsIds should come from the avs configurations, instead of hardcoding the values here. The actual values for the configurations will be provided via kcp chart values overrides coming from our config repo.

Reply:
understood. fixed.